classes/OccurrenceSearchSupport.php

```diff
@@ -1,4 +1,5 @@
 <?php
+include_once($SERVER_ROOT.'/content/lang/collections/misc/collstats.'.$LANG_TAG.'.php');
```

Comment:
Not sure what we should do about translations in class files. I guess the problem is the class file producing html in the first place, but seemed out of scope to solve right now, and it made sense.